platform/throughput-report/index.md

```diff
@@ -45,18 +48,11 @@ The containers required for generating a usage report are:
 - [RavenDB](/servicecontrol/ravendb/containers.md)
   - Alternatively another RavenDB source can be used. The [connection string](/servicecontrol/servicecontrol-instances/deployment/containers.md#required-settings-ravendb-connection-string) must be supplied when installing ServiceControl
 - [ServiceControl](/servicecontrol/servicecontrol-instances/deployment/containers.md)
-
-> [!WARNING]
-> By default, the ServiceControl instance will immediately start reading from the `error` queue when it starts. If this is not desired, e.g. if you have another process that monitors the error queue, then one of the following parameters should be supplied to the `docker run` command
->
-> - -e SERVICEBUS_ERRORQUEUE=<<name of non-existing queue, e.g. `errornotused`>>
-> - -e SERVICECONTROL_INGESTERRORMESSAGES=false
-
 - [ServicePulse](/servicepulse/containerization/)
 
 ### Cloud environments
 
-When hosting containers in the cloud, it is suggested to use [RavenDB Cloud](https://ravendb.net/cloud) to host the database. This is because the [storage requirements](https://ravendb.net/docs/article-page/6.2/csharp/start/installation/deployment-considerations#storage-considerations) of RavenDB generally cannot be met on most cloud offerings.
+When hosting containers in Kubernetes in any of the major Cloud providers, it is possible to host RavenDB in Kubernetes using the recommended storage providers by the Cloud infrastructure, see [these example manifests](https://github.com/Particular/PlatformContainerExamples/blob/main/helm/README.md#ravendb-deployment) for deployments in AKS or EKS. If the [storage requirements](https://ravendb.net/docs/article-page/6.2/csharp/start/installation/deployment-considerations#storage-considerations) of RavenDB cannot be met, it is suggested to use [RavenDB Cloud](https://ravendb.net/cloud) to host the database.
```

Comment:
I have expanded this section to mention Kubernetes, given that we don't need RavenDB Cloud in those hosts